network/README.md

```diff
@@ -30,6 +30,26 @@ This library is designed to be integrated in your program.
   h.ServeUnix("root", "test_network")
 ```
 
+### Example using Windows named pipes:
+
+```go
+import "github.com/docker/go-plugins-helpers/network"
+import "github.com/docker/go-plugins-helpers/sdk"
+
+d := MyNetworkDriver{}
+h := network.NewHandler(d)
+
+config := sdk.WindowsPipeConfig{
+  // open, read, write permissions for everyone 
+  // (uses Windows Security Descriptor Definition Language)
+  SecurityDescriptor: "S:(ML;;NW;;;LW)D:(A;;0x12019f;;;WD)",
```

Comment:
I haven't decoded this SDDL, but are you really sure you want (going by the comment) everyone access? What is the intended opener of this pipe? The docker daemon? (I'm not familiar with the plugin architecture or this project at all...) If so, it could be ACL'd to Local System and Administrators. Should non-admin users really have access?